### ISSUE TITLE
Compilation error for missing Components.Pills

### ISSUE THREAD

**Issue by jyeshe** [OpenFn/Lightning#1261]
Compilation error on release branch:
```
Compiling 96 files (.ex)
error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning/accounts/user_notifier.ex:6: Lightning.Accounts.UserNotifier (module)
error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/project_live/delete_connection_modal.ex:3: LightningWeb.ProjectLive.DeleteConnectionModal (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/job_live/credential_picker.ex:7: LightningWeb.JobLive.CredentialPicker (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/credential_live/form_component.ex:5: LightningWeb.CredentialLive.FormComponent (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/project_live/form_component.ex:13: LightningWeb.ProjectLive.FormComponent (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/project_live/settings.ex:6: LightningWeb.ProjectLive.Settings (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/workflow_live/edit.ex:3: LightningWeb.WorkflowLive.Edit (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/job_live/adaptor_picker.ex:6: LightningWeb.JobLive.AdaptorPicker (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/job_live/cron_setup_component.ex:15: LightningWeb.JobLive.CronSetupComponent (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/workflow_live/editor_pane.ex:2: LightningWeb.WorkflowLive.EditorPane (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/run_live/components.ex:3: LightningWeb.RunLive.Components (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/run_live/run_viewer_live.ex:2: LightningWeb.RunLive.RunViewerLive (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/components/form.ex:3: LightningWeb.Components.Form (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/run_live/workorder_component.ex:7: LightningWeb.RunLive.WorkOrderComponent (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/run_live/index.ex:5: LightningWeb.RunLive.Index (module)

error: module LightningWeb.Components.Pills is not loaded and could not be found. This may be happening because the module you are trying to load directly or indirectly depends on the current module
  lib/lightning_web/live/workflow_live/components.ex:3: LightningWeb.WorkflowLive.Components (module)


== Compilation error in file lib/lightning_web/live/credential_live/form_component.ex ==
** (CompileError) lib/lightning_web/live/credential_live/form_component.ex: cannot compile module LightningWeb.CredentialLive.FormComponent (errors have been logged)
    expanding macro: LightningWeb.__using__/1
    lib/lightning_web/live/credential_live/form_component.ex:5: LightningWeb.CredentialLive.FormComponent (module)
    (elixir 1.15.4) expanding macro: Kernel.use/2
would you have some idea at the top of your head? It happens on the manifest.json fix commit.
for now I am working 
```

on c5d15236ad92c814bc3cab44a2575ed28bb2ed9d (HEAD of `release-0.10.0`)

I have reproduced this locally on main:

- [ ] Yes
- [x] No

**To Reproduce** Steps to reproduce the behavior:

`$ mix do ecto.drop, ecto.setup`

**Comment by stuartc** [comment]
This should be fixed in 05cbc3821c6a78bc5630c0d5de7067286dbbe9cf

**Comment by jyeshe** [comment]
Fixed
